vector_search/tasks.py

```diff
@@ -254,3 +254,30 @@ def embed_new_learning_resources(self):
     embed_tasks = celery.group(tasks)
 
     return self.replace(embed_tasks)
+
+
+@app.task(bind=True)
+def embed_new_content_files(self):
+    """
+    Embed new content files from the last day
+    """
+    log.info("Running content file embedding task")
+    delta = datetime.timedelta(days=1)
+    since = now_in_utc() - delta
+    new_content_files = ContentFile.objects.filter(
+        published=True,
+        created_on__gt=since,
```

Comment:
should this be updated_on instead of created_on?

Reply:
That's a good question.  I was hoping to handle updates/deletion as a separate ticket since if I change this to updated_on it will generate completely new/duplicate embeddings since the hash ("key" field) of the file changes. The point ids are generated off of the key so I think we would have figure out how to delete the existing record and/or re-embed all content files for the particular course when a change like that happens

Reply:
placeholder ticket: https://github.com/mitodl/hq/issues/6754